Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/Dockerfile-trainer

```diff
@@ -1,6 +1,15 @@
 FROM python:3.7
 
+# Include GCP service account credentials with the build
+# Run docker build .... --build-arg GCP_CREDS_JSON_BASE64="xxxx"
+# where xxxx is a base64-encoded version of GCP creds JSON
+# The lab repo has a secret named GCP_CREDS_JSON_BASE64 with required data
+ARG GCP_CREDS_JSON_BASE64
+ENV GCP_CREDS_JSON_BASE64_ENV=${GCP_CREDS_JSON_BASE64}
+ENV GOOGLE_APPLICATION_CREDENTIALS=/root/gcp.json
+RUN echo "${GCP_CREDS_JSON_BASE64_ENV}"| base64 --decode > "${GOOGLE_APPLICATION_CREDENTIALS}"
+
 RUN pip install --upgrade pip && \
     pip install torch==1.7.1+cpu torchvision==0.8.2+cpu torchaudio==0.7.2 torchserve torch-model-archiver -f https://download.pytorch.org/whl/torch_stable.html && \
```

Comment:
I don't think this line is needed as `mlflow run` will use the included conda.yaml to build an environment then train.
Might be better in a different PR though.

Reply:
@savagej I tested your suggestion and it didn't work :(
mlflow either uses conda env OR docker env, so if we use docker we have to package it with all dependencies